.ruby-version

```diff
@@ -1 +1 @@
-2.3.1
+ruby-2.4.1
```

Comment:
How did the .ruby-version file even get here? You put it in git.ignore in (already merged) PR https://github.com/AgileVentures/shf-project/pull/273